.changeset/wild-dodos-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-dhis2': patch
+---
+
+Add warning for breaking versions
```

Comment:
I would be stronger about this and say "Removed support for DHIS2 v42"

I know we haven't really removed anything, but it's part of the messaging of "hey, don't use this adaptor version with API version 42 - and if you do, you're on your own"